diff -up devhelp-41.2/data/meson.build.bak devhelp-41.2/data/meson.build
--- devhelp-41.2/data/meson.build.bak	2022-06-22 09:28:43.615000000 +0000
+++ devhelp-41.2/data/meson.build	2022-06-22 09:29:10.215000000 +0000
@@ -19,7 +19,6 @@ appdata_conf = configuration_data()
 appdata_conf.set('application_id', APPLICATION_ID)
 appdata = APPLICATION_ID + '.appdata.xml'
 appdata_file = I18N.merge_file(
-  appdata,
   input: configure_file(
     input: 'org.gnome.Devhelp.appdata.xml.in.in',
     output: APPLICATION_ID + '.appdata.xml.in',
@@ -45,7 +44,6 @@ desktop_conf = configuration_data()
 desktop_conf.set('application_id', APPLICATION_ID)
 desktop = APPLICATION_ID + '.desktop'
 desktop_file = I18N.merge_file(
-  desktop,
   type: 'desktop',
   input: configure_file(
     input: 'org.gnome.Devhelp.desktop.in.in',
diff -up devhelp-41.2/plugins/gedit-plugin/meson.build.bak devhelp-41.2/plugins/gedit-plugin/meson.build
--- devhelp-41.2/plugins/gedit-plugin/meson.build.bak	2022-06-22 09:29:22.208000000 +0000
+++ devhelp-41.2/plugins/gedit-plugin/meson.build	2022-06-22 09:29:32.582000000 +0000
@@ -7,7 +7,6 @@ install_data(
 
 plugin_info_file = 'devhelp.plugin'
 I18N.merge_file(
-  plugin_info_file,
   type: 'desktop',
   input: plugin_info_file + '.desktop.in',
   output: plugin_info_file,
